mmdet/models/dense_heads/yolo_head.py

```diff
@@ -0,0 +1,652 @@
+                [( 30, 61), ( 62,  45), ( 59, 119)],
+                [( 10, 13), ( 16,  30), ( 33,  23)]]
+        ignore_thresh (float): Set negative samples if gt-anchor iou
+            is smaller than ignore_thresh. Default: 0.5
+        one_hot_smoother (float): Set a non-zero value to enable label-smooth
+            Default: 0.
+        xy_use_logit (bool): Use log scale regression for bbox center
+            Default: False
+        balance_conf (bool): Whether to balance the confidence when calculating
+            loss. Default: False
+        conv_cfg (dict): Config dict for convolution layer. Default: None.
+        norm_cfg (dict): Dictionary to construct and config norm layer.
+            Default: dict(type='BN', requires_grad=True)
+        act_cfg (dict): Config dict for activation layer.
+            Default: dict(type='LeakyReLU', negative_slope=0.1).
+        train_cfg (dict): Training config of YOLOV3 head. Default: None.
+        test_cfg (dict): Testing config of YOLOV3 head. Default: None.
+    """
+
+    def __init__(self,
+                 num_classes,
+                 num_scales,
+                 num_anchors_per_scale,
+                 in_channels,
+                 out_channels,
+                 strides,
+                 anchor_base_sizes,
+                 ignore_thresh=0.5,
+                 one_hot_smoother=0.,
+                 xy_use_logit=False,
+                 balance_conf=False,
+                 conv_cfg=None,
+                 norm_cfg=dict(type='BN', requires_grad=True),
+                 act_cfg=dict(type='LeakyReLU', negative_slope=0.1),
+                 train_cfg=None,
+                 test_cfg=None):
+        super(YOLOV3Head, self).__init__()
+        # Check params
+        assert (num_scales == len(in_channels) == len(out_channels) ==
+                len(strides) == len(anchor_base_sizes))
+        for anchor_base_size in anchor_base_sizes:
+            assert (len(anchor_base_size) == num_anchors_per_scale)
+            for anchor_size in anchor_base_size:
+                assert (len(anchor_size) == 2)
+
+        self.num_classes = num_classes
+        self.num_scales = num_scales
+        self.num_anchors_per_scale = num_anchors_per_scale
+        self.in_channels = in_channels
+        self.out_channels = out_channels
+        self.strides = strides
+        self.anchor_base_sizes = anchor_base_sizes
+
+        self.ignore_thresh = ignore_thresh
+        self.one_hot_smoother = one_hot_smoother
+        self.xy_use_logit = xy_use_logit
+        self.balance_conf = balance_conf
+
+        # self.num_attrib = num_classes + bboxes (4) + objectness (1)
+        self.num_attrib = self.num_classes + 5
+        self.last_layer_dim = self.num_anchors_per_scale * self.num_attrib
+
+        cfg = dict(conv_cfg=conv_cfg, norm_cfg=norm_cfg, act_cfg=act_cfg)
+        self.convs_bridge = nn.ModuleList()
+        self.convs_final = nn.ModuleList()
```

Comment:
`convs_final` -> `convs_pred`

Reply:
Sure. But, changing the name will introduce a conversion of pretrained models.

Reply:
I will take care of that.

Reply:
Thanks @xvjiarui . `convs_final` -> `convs_pred` and `conv_final` -> `conv_pred`.